library/silverpeas

```diff
@@ -1,6 +1,9 @@
+# This file is generated via https://github.com/Silverpeas/docker-silverpeas-prod/blob/master/generate-docker-library.sh
 Maintainers: Miguel Moquillon <miguel.moquillon@silverpeas.org> (@mmoqui)
 GitRepo: https://github.com/Silverpeas/docker-silverpeas-prod.git
 
-Tags: 6.0, latest
-GitCommit: 80bd9752fac5793594ddc80943ecdcf88e25f3b1
+Tags: 6.0.1, latest
+GitCommit: 43daef47765ff3acc21c4d856a0bc3e09f2e9de3
 
+Tags: 6.0
+GitCommit: 80bd9752fac5793594ddc80943ecdcf88e25f3b1
```

Comment:
Any particular reason to keep this version listed here?  Removing it here will remove it from the "Supported" section on the Hub readme (and will prevent us from spending cycles rebuilding it on the official build server), but the tag will still be available to users who want it.  (See https://github.com/docker-library/official-images#library-definition-files for more detail on this.)

Reply:
No peculiar reasons if it is still available after being removed.
I removed this tag.